src/services/clients/dissolutionApi.client.ts

```diff
@@ -0,0 +1,33 @@
+import 'reflect-metadata'
+
+import { AxiosInstance } from 'axios'
+import { inject } from 'inversify'
+import { provide } from 'inversify-binding-decorators'
+
+import { DissolutionCreateRequest } from 'app/models/dto/dissolutionCreateRequest'
+import { DissolutionCreateResponse } from 'app/models/dto/dissolutionCreateResponse'
+import Optional from 'app/models/optional'
+import TYPES from 'app/types'
+
+@provide(DissolutionApiClient)
+export class DissolutionApiClient {
+  public constructor(
+    @inject(TYPES.DISSOLUTIONS_API_URL) private DISSOLUTIONS_API_URL: Optional<string>,
+    @inject(TYPES.AxiosInstance) private axios: AxiosInstance
+  ) {}
+
+  public async createDissolution(token: string, companyNumber: string,
+                                 body: DissolutionCreateRequest): Promise<DissolutionCreateResponse> {
```

Comment:
How is this compiling? I don;t see you doing `response.data` anywhere? I would expect:
```
const response: AxiosResponse<DissolutionCreateResponse> = await this.axios.post(...)
return response.data
```